### ISSUE TITLE
Output contains droplets with 0 counts 

### ISSUE THREAD

**Issue by henryjtaylor** [broadinstitute/CellBender#63]
Hi,

I am running CellBender on UMI-tagged scRNA data. To run it, I am using the following command:

```
cellbender remove-background \
            --input ${input_files} \
            --output ${outfile} \
            --cuda \
            --expected-cells ${expected_cells} \
            --total-droplets-included ${total_droplets_include} \
            --model full \
            --z-dim 200 \
            --z-layers 1000 \
            --low-count-threshold 10 \
            --epochs ${epochs} \
            --empty-drop-training-fraction 0.5 \
            --learning-rate ${learning_rate}
```

The output generally looks good. Below I have a few plots describing the model and cell probabilities. 

![image](https://user-images.githubusercontent.com/17941308/87686469-858bd180-c752-11ea-8237-d4aeb0cfef5d.png)

However, there are 2 'cells' included that have 0 counts in the filtered output. It's very easy to deal with, but seems like weird behavior given the application.

**Comment by mbabadi** [comment]
Dear @henryjtaylor,

Thank you for using CellBender and reporting back!  yes, indeed that's an undesirable behavior.  My best explanation is that the algorithm has identified a likely non-empty droplet, however, after removing the background portion, the counts have gone down to 0. This is quite a rare phenomenon -- good catch! we will make sure to check for such edge cases in the next release.

**Comment by SNRNS** [comment]
I also see the same and in some cases, quite many obs with 0 counts.  Though it seems to be proportional to the total number of observations cell bender labels as real cells. Either way is strange to see this in the output.

![image](https://user-images.githubusercontent.com/14887981/120034675-69d89500-bfcb-11eb-97b8-5158da1a6a78.png)

**Comment by BradBalderson** [comment]
I also see this, and does not appear to be that rare in my dataset:

![image](https://github.com/broadinstitute/CellBender/assets/24308582/f0c67b86-eb4a-4deb-a489-c7dd8adebb44)

Many low-count cells here, the 10th quantile is 106UMIs, which is very low.
